doc/api.md

```diff
@@ -182,19 +201,33 @@ used should be taken from those defined by
 }
 ```
 
-<!-- END OF TOPICS WIP -->
+## Securized access
+
+**trust** token to use for secured access to the Context Broker for this type of devices (optional; only needed for
+secured scenarios). Trust tokens may be called `access_tokens` by some Oauth2 providers.
+
```

Comment:
This section is part of advanced topics file. This section was created with information present in config group access. Since there is a section more detailed, it is better to delete this section in this PR